src/pages/home/_Banner.tsx

```diff
@@ -0,0 +1,36 @@
+import { FunctionComponent } from 'react'
+
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import Link from 'next/link'
+
+import { buttonLocation, buttonStyle } from '@data'
+
+const Banner: FunctionComponent = () => (
+    <div className="sg-bg-gradient-starship tw-px-sm tw-pb-sm lg:tw-pb-0 xl:tw-max-h-5xl">
+        <div className="tw-max-w-screen-lg tw-mx-auto tw-flex tw-flex-col lg:tw-flex-row tw-justify-around tw-items-center">
+            <div className="tw-h-[80px]">
```

Comment:
Neat 🤓, TW FTW 🙌🏻 !  Should we fix this asset to have the right dimensions so the container element doesn't need to overwrite the height? Or later on? Not sure how purposeful this is, or if we're just trying to move fast to ship it.

Reply:
The reason I'm clipping this with a fixed height is so we can reuse the full size SVG for other use cases without changing the viewbox/width/height within the SVG or having dupe assets to accomodate different use cases. Since the shadow on "Starship" gets clipped within the banner in the Figma, this was a trick around that. 🤓

Reply:
Gotcha! Thanks for the context 😄  -- Maybe something we can bring into our DLS sync, but is there a standard we want to abide by for external white space around assets/svgs? Like this one coming in has some extra top & right spacing. To me, I'd think we want the svg content to be sitting flesh to it's borders in most cases. What do you think?

Reply:
I agree! I think that should be the standard so we can bring this up during our DLS sync. Would love to have each asset fit flush within their viewbox.